test/python/sdpa/frost/test_sdpa_fwd_split_kv_sm120.py

```
@@ -0,0 +1,143 @@
+    torch.manual_seed(0)
+    q = torch.randn(b, 8, 512, d, device=dev, dtype=torch.float16)
+    k = torch.randn(b, 1, 8192, d, device=dev, dtype=torch.float16)
+    v = torch.randn(b, 1, 8192, d, device=dev, dtype=torch.float16)
+    o = torch.zeros_like(q)
+
+    bad = SdpaFwdDslSm120(sample_q=q, sample_k=k, sample_v=v, sample_o=o, is_causal=True, split_kv=2)
+    assert bad.check_support()
+    with pytest.raises(ValueError, match="split_kv"):
+        bad.compile()  # derived LPT + split: the config backstop rejects
+
+    split, got, ref, _, _expected = _sm120_case(8, 1, 512, 8192, causal=True)
```

Ruff (0.16.1)

[warning] Unpacked variable `split` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)